programs/local/LocalServer.cpp

```diff
@@ -412,6 +412,14 @@ void LocalServer::connect()
 }
 
 
+void LocalServer::prepareForInteractive()
+{
+    clearTerminal();
+    showClientVersion();
+    std::cerr << std::endl;
```

Comment:
Why do we print new line only in `clickhouse-local`?

Reply:
Because for client there is one more line afterwards and it will print new line:
https://github.com/ClickHouse/ClickHouse/blob/4c675f0151cda5ed79b6f57412126f581829d98d/programs/client/Client.cpp#L557-L559